### DISCUSSION TITLE
Failed commit due to permission CreateCommitOnBranch

### DISCUSSION THREAD

**Discussion by azro352** [community/community#141693] [Mobile] [answered]
I have a project where I successfully edited the readme and commited on the mobile app

I tried to do same on `.github/workflows/wf.yaml` but I got the following error when trying to commit

![image](https://github.com/user-attachments/assets/63cb3017-4cd4-4a5a-9c39-3e9c264414ab)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.181.0 (143346435)
Device: iPhone SE (2nd generation) (iOS 18.0.1)
</details>

**Comment by stevepopovich** [comment]
@azro352 thanks for writing in! So, you can edit that workflow on the website fine, it just doesn't work on mobile. Is that correct?

**Reply by azro352** [reply]
That is correct

**Reply by stevepopovich** [reply]
Interesting. We are having trouble reproducing. Are you an admin of this repository?

**Reply by azro352** [reply]
Yes, I'm the owner 

Here's a video of it, it doesn't add details to this story but to be sure the issue is well understood

https://github.com/user-attachments/assets/ed01ad3d-d6a4-4529-b6d9-d873b48f8214

**Reply by stevepopovich** [reply]
wow, ok. That is great, thank you very much for adding this video.

**Reply by stevepopovich** [reply]
Do you have any branch protection rules on that repo? i.e. anything that prevent you from committing directly to `master`?

**Reply by azro352** [reply]
No

On the app I just:

- successfully created and modified a file at root level
- successfully created and modified a file in the .github directory 
- I have failed, with the same error, to create a file in the .github/workflows directory

**Reply by stevepopovich** [reply]
Wow, very interesting. Ok, let me do some investigation, thank you!

**Reply by azro352** [reply]
@stevepopovich any updates ? :)

**Reply by stevepopovich** [reply, marked as answer]
At this time, we still can't repro. Are you able to log in and out? We think this could help.

**Reply by azro352** [reply]
Log in and out worked... 🎉

**Reply by stevepopovich** [reply]
ah great. Thank you!

**Comment by HIMANSHUTHAKUR00** [comment]
great
